src/ui/OpenFile/UiOpenMenu.jsx

```diff
@@ -18,11 +18,920 @@ import { UiReportMenu } from './UiReportMenu';
 import { Tooltip } from '../Tooltip/Tooltip';
 import { Container } from '../Layout/Container';
 import FileReader from '../../engine/loaders/FileReader';
+import StoreActionType from '../../store/ActionTypes';
 
 /** Need to have demo menu */
 const NEED_DEMO_MENU = true;
 
-class UiOpenMenu extends FileReader {
```

Comment:
please be careful with this component, now it extends from FileReader

Reply:
@oleksandr-zhynzher, sorry, I guess while the merge was done I accidentally removed this part. But isn't it a bad approach though? https://legacy.reactjs.org/docs/composition-vs-inheritance.html